draft-ietf-jsonpath-base.md

```diff
@@ -159,6 +159,20 @@ in ABNF as `%x2318`.
 
 The terminology of {{-json}} applies.
 
+For the purposes of this specification, a JSON value as defined by
+{{-json}} is also viewed as a tree of nodes.
```

Comment:
This is good.  We don't need to claim that JSON itself is a tree of nodes, only that this spec views them as such when convenient.

Maybe instead of "is also viewed..." we use "can also be viewed..."

Reply:
Well, we very actively view it as a tree of nodes in the current text, so I think we are good as is.

Reply:
I like that "view" trick ...